### ISSUE TITLE
Remove extra, old records showing up in certain projects' activity log

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#9642]
In, at least, project ID 227 & 228, there are activity log entries which need to be removed.

![Screen Shot 2022-07-01 at 2 13 26 PM](https://user-images.githubusercontent.com/3653206/176963128-c20a3858-3ba6-42b0-8105-b139e11701b1.png)

They appear to be dated at or around when we launched Moped. Please see this thread in slack for more information: https://austininnovation.slack.com/archives/CNUEPKLB1/p1656702937771599.

Observations:
1) On these projects, there is inconsistent use of UUIDs to link to the `moped_users` table
1) Getting the records using the wrong UUID source out may be enough to clean up this problem.
1) To know this for sure,the records in the database need to be consider in context with the code driving the activity log rendering. This will hopefully make clear what data belongs where. 

As a spin-off issue, I think this work will reveal that we have meaningful opportunities to normalize the activity log feature in the moped application.

**Comment by frankhereford** [comment]
Here are some sql snippets that i used looking at the read replica that might be helpful:

```


select log.activity_id, log.record_project_id, log.created_at, log.description, projects.project_id
from moped_activity_log log
left join moped_project projects on (log.record_project_id = projects.project_id)
--where projects.project_id is not null
order by log.record_id desc, log.created_at asc;

select count(*), record_project_id
from moped_activity_log log
group by record_project_id 
order by record_project_id desc

select max(project_id) from moped_project;



select updated_by, description  
from moped_activity_log
where true
and 
(
  updated_by = 'e973f569-645c-4e34-b08d-c2f36e9a1ae6' -- amenity's moped_users.staff_uuid
    or
  updated_by = '23bb4e0c-0658-4148-8d63-872f1ecb539c' -- amenity's moped_users.cognito_user_id 
)
and record_project_id = 228
order by created_at desc;



select * from moped_users where first_name = 'Amenity';
```

**Comment by johnclary** [comment]
just want to point a tiny potential bug in the [activity log query](https://github.com/cityofaustin/atd-moped/blob/f19ae38ce0079ddabb158d5f1df4c9031118f1f1/moped-editor/src/queries/project.js#L415), we're filtering for active users only

```
moped_users(where: { status_id: { _eq: 1 } }) 
```

we don't want to do that—a no-longer-active user may be responsible for historical project activities.

**Comment by amenity** [comment]
Thanks, @frankhereford @chiaberry @johnclary. Would anyone care to hazard an estimate for this?

**Comment by chiaberry** [comment]
I can't tell if its a straightforward removing data that predates a certain date (then I would say 3) or if it would be more complex (5)

**Comment by amenity** [comment]
Could be done with #9726

**Comment by johnclary** [comment]
Thanks to Frank's queries—I identified 11 records that were clearly created before the migration and I deleted them with this

```sql 
delete from moped_activity_log where created_at < '2022-02-11' and (record_project_id = 228 or record_project_id = 227);
```

After further research, there is bad data in our users table: Renee's congnito UUID is the same as Amenity's staff UUID. This caused us to wonder if 

> there is inconsistent use of UUIDs to link to the moped_users table

Fortunately, that does not appear to be the case. I will open a separate issue.
